flow/util/genMetrics.py

```diff
@@ -251,16 +251,10 @@ def extract_metrics(cwd, platform, design, flow_variant, output, hier_json):
 
     hierarchicalSynth = os.environ.get('SYNTH_HIERARCHICAL')
```

Comment:
This is no longer needed, it was only used in the if/else statement. Could you remove it?